brokerapi/brokers/cloudsql/mysql-definition.go

```diff
@@ -173,10 +179,70 @@ func init() {
 				    }
 				]
 		}`,
-		ProvisionInputVariables: commonProvisionVariables,
-		DefaultRoleWhitelist:    roleWhitelist,
-		BindInputVariables:      commonBindVariables,
-		BindOutputVariables:     commonBindOutputVariables,
+		ProvisionInputVariables: append([]broker.BrokerVariable{
+			{
+				FieldName: "instance_name",
+				Type:      broker.JsonTypeString,
+				Details:   "Name of the Cloud SQL instance.",
+				Default:   "pcf-sb-${counter.next()}-${time.nano()}",
+				Constraints: validation.NewConstraintBuilder().
+					Pattern("^[a-z][a-z0-9-]+$").
+					MaxLength(84).
+					Build(),
+			},
+			{
+				FieldName: "database_name",
+				Type:      broker.JsonTypeString,
+				Details:   "Name of the database inside of the instance. Must be a valid identifier for your chosen database type.",
+				Default:   "pcf-sb-${counter.next()}-${time.nano()}",
```

Comment:
I'm leaving the validation for these blank because this is implementation specific.